build.sh

```diff
@@ -16,7 +16,7 @@ set -e
 TOPDIR="$(pwd)"
 
 PREFIX_BUILD="$(pwd)/_build/$TARGET"
-PREFIX_BUILD_HOST="$(pwd)/_build/host"
+PREFIX_BUILD_HOST="$(pwd)/_build/host-pc"
```

Comment:
True. I forgot about it. 
Now I'am prepared a change, but need the submodule commits to update submodules along with change. If I not do it this way it will break build.

Reply:
ok, I'm merging the PRs now.